### DISCUSSION TITLE
List changed files

### DISCUSSION THREAD

**Discussion by KeyboardSounds** [abraunegg/onedrive#2188] [Q&A] [answered]
Is there a way to see a list of which files have changed before performing a `--synchronize`?

`--display-sync-status` only seems to show the change in size, even with the verbose flag set.

**Comment by abraunegg** [comment, marked as answer]
@KeyboardSounds 
> Is there a way to see a list of which files have changed before performing a --synchronize?

Not directly, however if you use `--synchronize --dry-run` that will give you what will happen on the next sync - ie .. what has changed.
